src/steps/vulnerabilities/index.ts

```diff
@@ -75,6 +77,12 @@ export async function fetchAssets(
       timeoutInMinutes: assetApiTimeoutInMinutes,
     },
   );
+
+  if (duplicateKeysEncountered > 0) {
+    logger.info(
+      `Found ${duplicateKeysEncountered} duplicate keys for "tenable_asset" entity`,
```

Comment:
Let's do this one like
```suggestion
       { duplicateKeysEncountered }, 
      `Found duplicate keys for "tenable_asset" entity`,
```